deposit/src/main/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJob.java

```diff
@@ -171,7 +173,7 @@ public void runJob() {
 		// Ingest objects included in this deposit into the destination object
 		try {
 			ingestChildren((ContentContainerObject) destObj, depositBag);
-		} catch (DepositException | FedoraException e) {
+		} catch (DepositException | FedoraException | IOException e) {
```

Comment:
One thing I noticed is that it seems like the IOException from addDescription will never make it back to this point.  It's immediately caught and rethrown as a transaction interruption exception everywhere except in addFileObject, which does the same one step up in the addWork* method that calls it.  So you may be able to skip having `throws IOException` in a lot of the cases added.